src/open_inwoner/js/components/primary-navigation/index.js

```diff
@@ -0,0 +1,24 @@
+const primaryNavItems = document.querySelectorAll(
+  '.primary-navigation .primary-navigation__list'
```

Comment:
I'm assuming all of this could be deleted due to the new "_src/open_inwoner/js/components/header/primary-navigation.jssrc/open_inwoner/js/components/header/primary-navigation.js_" file?

Reply:
Yes, indeed this will be removed as well.